packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -325,6 +327,16 @@ class NavigationRail extends StatefulWidget {
   /// The default value is 256.
   final double? minExtendedWidth;
 
+  /// The curve for animations of the [NavigationRail].
```

Comment:
It would be helpful to clarify what these animations are (here and in the duration API doc).